pkg/controllers/provisioningv2/fleetcluster/fleetcluster.go

```diff
@@ -28,6 +28,10 @@ import (
 	"k8s.io/client-go/tools/clientcmd"
 )
 
+const (
+	turtlesOwnedLab = "cluster-api.cattle.io/owned"
```

Comment:
```suggestion
	externallyManagedAnn  = "provisioning.cattle.io/externally-managed"
```

I think this is better suited as an annotation